spec/models/post.spec.js

```diff
@@ -37,4 +37,26 @@ describe("Post model", () => {
       });
     });
   });
+
+  it("displays posts by most recent", (done) => {
+    var post1 = new Post({ message: "some message" });
+
+    post1.save((err) => {
+      expect(err).toBeNull();
+    });
+
+    var post2 = new Post({ message: "some different message" });
+
+    post2.save((err) => {
+      expect(err).toBeNull();
+      console.log('here2');
+
+      Post.find({}).sort({date: -1}).exec((err, posts) => {
```

Comment:
Also logic bang on, but bare in mind this is the unit tests for posts in the model, ideally here we should be checking if the date is being saved when you save a post